src/Switch/Switch.css

```diff
@@ -0,0 +1,111 @@
+.toggle-switch {
```

Comment:
I have pulled the switch into the little example app in this repo and it doesn't pull the styles through

Reply:
Fixed